src/forms/radio-group.scss

```diff
@@ -0,0 +1,40 @@
+// Tab like radio group
+
+.radio-group {
+  @include clearfix;
+}
+
+.radio-label {
+  float: left;
+  padding: $spacer-2 10px $spacer-2 35px;
+  // stylelint-disable-next-line primer/spacing
+  margin-left: -1px;
+  color: $text-gray-dark;
+  cursor: pointer;
+  border: $border-width $border-style $border-gray-dark;
+
+  :checked + & {
+    position: relative;
+    z-index: 1;
+    border-color: $border-blue;
+  }
+
+  &:first-of-type {
+    margin-left: 0;
+    border-top-left-radius: $border-radius;
+    border-bottom-left-radius: $border-radius;
+  }
+
+  &:last-of-type {
+    padding-right: $spacer-3;
+    border-top-right-radius: $border-radius;
+    border-bottom-right-radius: $border-radius;
+  }
+}
+
+.radio-input {
+  z-index: 3;
+  float: left;
+  // stylelint-disable-next-line primer/spacing
+  margin: 14px -35px 0 14px;
```

Comment:
Top margin is a bit too much:

![Screen Shot 2019-11-18 at 11 00 46 AM](https://user-images.githubusercontent.com/378023/69019820-99313d80-09f5-11ea-9873-1feb54537550.png)

Maybe something we also could look into: Have the same height as other inputs/buttons. So that it matches when shown inline. I'll try that.